buildNumber.properties

```diff
@@ -1,3 +1,3 @@
 #maven.buildNumber.plugin properties file
```

Comment:
Can you drop this commit? We only dump these things when releasing.

Reply:
revert done